website/static/js/koHelpers.js

```diff
@@ -209,6 +209,46 @@ ko.bindingHandlers.fadeVisible = {
     }
 };
 
+var fitHelper = function(value, length, replacement, trimWhere) {
```

Comment:
If one or both of these are the code we're using from Punches, we should label it as such.